From cff9887d760d3678198a3b8de8602cf464eb2f31 Mon Sep 17 00:00:00 2001
From: Denny Harijanto <denny@nusantara-cloud.com>
Date: Sun, 3 Feb 2019 11:36:15 +0700
Subject: [PATCH 1/5] Revert "Update TF101"

This cause Android USB networking to fail.

This reverts commit 1a5e680a26c7f820a60bea26ace5d1c014e4f8b3.
---
 drivers/usb/gadget/f_rndis.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/usb/gadget/f_rndis.c b/drivers/usb/gadget/f_rndis.c
index 3e3dcf401c8..f187b9eee9e 100644
--- a/drivers/usb/gadget/f_rndis.c
+++ b/drivers/usb/gadget/f_rndis.c
@@ -857,8 +857,8 @@ static inline bool can_support_rndis(struct usb_configuration *c)
  * for calling @gether_cleanup() before module unload.
  */
 int
-rndis_bind_config(struct usb_configuration *c, u8 ethaddr[ETH_ALEN]/*,
-				u32 vendorID, const char *manufacturer*/)
+rndis_bind_config(struct usb_configuration *c, u8 ethaddr[ETH_ALEN],
+				u32 vendorID, const char *manufacturer)
 {
 	struct f_rndis	*rndis;
 	int		status;
@@ -903,8 +903,8 @@ rndis_bind_config(struct usb_configuration *c, u8 ethaddr[ETH_ALEN]/*,
 		goto fail;
 
 	memcpy(rndis->ethaddr, ethaddr, ETH_ALEN);
-//	rndis->vendorID = vendorID;
-//	rndis->manufacturer = manufacturer;
+	rndis->vendorID = vendorID;
+	rndis->manufacturer = manufacturer;
 
 	/* RNDIS activates when the host changes this filter */
 	rndis->port.cdc_filter = 0;
-- 
2.19.1

